packages/gafl-webapp-service/src/__mocks__/data/defra-country.js

```diff
@@ -11,6 +11,11 @@ export default {
       label: 'United Kingdom',
       description: 'GB'
```

Comment:
Should United Kingdom above now be removed?

Reply:
This data is used by quite a few tests, so removing this would break other tests. More specifically, the address lookup sets the country to 'United Kingdom', so it needs to be available in the master list, or it breaks other parts of the code.